Pixel Game/Assets/Scripts/Movement.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    private const float KnockbackAcceleration = 0.5f;
+    [SerializeField]
+    protected float maxSpeed = 5;
+    [SerializeField]
+    private float acceleration = 5;
+    [SerializeField]
+    protected Animator animator;
+
+    private Health health;
+    protected Vector2 movementDir;
+    protected Vector2 faceDir = Vector3.zero;
+    private Rigidbody2D body;
+    
+    //Don't you just love properties and ternary operators? I do.
+    private float EffectiveAcceleration =>
+        health != null && health.KnockedBack ?
+        KnockbackAcceleration : acceleration;
+
+    //public float bootsSpeedGang;          // Use when equipment is being added
```

Comment:
We can remove these boots variables, since we wont be using visual equipment

Reply:
Right, will do.